client/state/site-settings/schema.js

```diff
@@ -0,0 +1,10 @@
+export const items = {
+	type: 'object',
+	additionalProperties: false,
+	patternProperties: {
+		// Site Id
+		'^\\d+$': {
+			type: 'object'
```

Comment:
Maybe in the future, but it would be a good idea to add validation for properties of the settings endpoint response we're using in Calypso. Doesn't need to be all-encompassing, but would help protect against changes or unexpected value types.